packages/build/tests/core/tests.js

```diff
@@ -227,3 +227,15 @@ if (platform !== 'win32') {
 
   const PID_LINE_REGEXP = /^PID: (\d+)$/m
 }
+
+test('Functions config is passed to zip-it-and-ship-it (1)', async (t) => {
```

Comment:
These unimaginative fixture names come from the fact that the three fixtures are identical — all of them require three Node modules, we're just changing which of those will be flagged as external modules and how.